Index: src/use_case/nextturn/NextTurnInteractor.java
===================================================================
diff --git a/src/use_case/nextturn/NextTurnInteractor.java b/src/use_case/nextturn/NextTurnInteractor.java
deleted file mode 100644
--- a/src/use_case/nextturn/NextTurnInteractor.java	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
+++ /dev/null	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
@@ -1,4 +0,0 @@
-package use_case.nextturn;
-
-public class NextTurnInteractor {
-}
Index: src/use_case/nextturn/NextTurnInputDataBoundary.java
===================================================================
diff --git a/src/use_case/nextturn/NextTurnInputDataBoundary.java b/src/use_case/nextturn/NextTurnInputDataBoundary.java
deleted file mode 100644
--- a/src/use_case/nextturn/NextTurnInputDataBoundary.java	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
+++ /dev/null	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
@@ -1,4 +0,0 @@
-package use_case.nextturn;
-
-public interface NextTurnInputDataBoundary {
-}
Index: src/use_case/nextturn/NextTurnInputData.java
===================================================================
diff --git a/src/use_case/nextturn/NextTurnInputData.java b/src/use_case/nextturn/NextTurnInputData.java
deleted file mode 100644
--- a/src/use_case/nextturn/NextTurnInputData.java	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
+++ /dev/null	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
@@ -1,9 +0,0 @@
-package use_case.nextturn;
-
-public class NextTurnInputData {
-    public String current_player;
-
-    public NextTurnInputData(String Current_player){
-        this.current_player = Current_player;
-    }
-}
Index: src/use_case/nextturn/NextTurnDataAccessInterface.java
===================================================================
diff --git a/src/use_case/nextturn/NextTurnDataAccessInterface.java b/src/use_case/nextturn/NextTurnDataAccessInterface.java
deleted file mode 100644
--- a/src/use_case/nextturn/NextTurnDataAccessInterface.java	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
+++ /dev/null	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
@@ -1,4 +0,0 @@
-package use_case.nextturn;
-
-public interface NextTurnDataAccessInterface {
-}
Index: src/use_case/nextturn/NextTurnOutputData.java
===================================================================
diff --git a/src/use_case/nextturn/NextTurnOutputData.java b/src/use_case/nextturn/NextTurnOutputData.java
deleted file mode 100644
--- a/src/use_case/nextturn/NextTurnOutputData.java	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
+++ /dev/null	(revision c87aad4e9d156dfe3e2ad04838ab92d471958bbf)
@@ -1,4 +0,0 @@
-package use_case.nextturn;
-
-public class NextTurnOutputData {
-}
